src/components/shared/inputs/DateInput.tsx

```diff
@@ -1,15 +1,17 @@
-import { ACTIVATION_DATE, CANCELLATION_DATE, DATE } from 'lib/messageTypes'
+import { ActionMap, Container } from 'constate'
+import { DATE } from 'lib/messageTypes'
 import * as moment from 'moment'
 import 'moment/locale/sv'
-import * as PropTypes from 'prop-types'
 import * as React from 'react'
 import { SingleDatePicker } from 'react-dates'
 import { OPEN_UP } from 'react-dates/constants'
 import 'react-dates/initialize'
+import styled from 'react-emotion'
+import { Mount } from 'react-lifecycle-components'
 import { Form } from 'semantic-ui-react'
-import styled from 'styled-components'
+import { dateInputStyles } from './_dateinput'
 
-const WidgetContainer = styled.div`
+const WidgetContainer = styled('div')`
```

Comment:
We prefer the object-based syntax: 
```suggestion
const WidgetContainer = styled('div')({
  display: 'flex',
```